packages/attestation-service/src/sms.ts

```diff
@@ -0,0 +1,74 @@
+import { PhoneNumberUtil } from 'google-libphonenumber'
+import Nexmo from 'nexmo'
+import { fetchEnv } from './env'
+
+const phoneUtil = PhoneNumberUtil.getInstance()
+
+let nexmoClient: any
```

Comment:
can we type this?

Reply:
The library unfortunately does not have any types

Reply:
You can define you own but not important for now